app/Http/Controllers/Auth/EditProfileController.php

```diff
@@ -113,4 +113,15 @@ public function edit(Request $request)
 
         return;
     }
-}
\ No newline at end of file
+
+    public function editpassword(Request $request)
+    {
+        $user = User::find($request->route('user'));
+
+        $user->password = Hash::make($request->new_password);
```

Comment:
Make sure to pass the request through a backend validator as well, just check that it's required, a string and at least 8 characters long.